psutil/_psosx.py

```diff
@@ -212,6 +214,31 @@ def disk_partitions(all=False):
 # =====================================================================
 
 
+def sensors_temperatures():
+    """Return hardware (CPU and others) temperatures as a dict
+    including hardware name and temperature.
+    """
+    ret = collections.defaultdict(list)
+    rawdict = cext.sensors_temperatures()
+
+    for name, values in rawdict.items():
+        current = round(values['current'], 2)
+        label = ''
+        high = values['high']
+        if high == int(0):
+            high = 105
+        critical = None
+
+        if high and not critical:
+            critical = high
+        elif critical and not high:
+            high = critical
```

Comment:
since you can't retrieve critical from C then you could just do `critical = high`

Reply:
Sure. Copied this from _pslinux.py